plugins/module_utils/http.py

```diff
@@ -33,6 +33,8 @@ def json(self):
         if self._json is None:
             try:
                 self._json = json.loads(self.data)
+            except TypeError: # Handle python 3.5 returning bytes
+                self._json = json.loads(self.data.decode('utf-8'))
```

Comment:
Would change block with code below, since we can fail in second case to. And also plese rebase to latest master, which fixes sanity tests.

```
            except TypeError:  # Handle python 3.5 returning bytes
                try:
                    self._json = json.loads(self.data.decode('utf-8'))
                except ValueError:
                    self._json = None
```

Reply:
@juremedvesek Thank you for your feedback. The pull request is rebased and updated.